packages/math/init.lua

```diff
@@ -353,6 +353,16 @@ Finally, here is a little secret. This notation:
 
 \noindent In other words, the notation using \code{&} and \code{\\\\} is only a syntactic sugar for a two-dimensional array constructed with braces.
 
+\paragraph{Equations numbering}
+Equations can be numbered in display mode adding the options \code{numbering} and \code{counter}.
```

Comment:
`\autodoc:parameter{numbering]` etc.
And possibly `\autodoc:parameter{counter=<id>}` a bit below.
(This gets syntax coloring if enable, and one day I hope we'll subscribe to a revamped indexer too)

Reply:
Thanks for the sugestion. In this regard, I'll also change `\equation` and `\math`, below, to use `autodoc:command{..}`.

Reply:
BTW, I haven't tested, but with math being a reserved keyword (see other command), I think that `\autodoc:command` might have troubles with it (or not do behaves as intended)